chordspy/app.py

```
@@ -0,0 +1,369 @@
+    # Create new connection
+    connection_manager = Connection()
+    
+    def run_connection():
+        """
+        Internal function to handle the connection process in a thread.
+        """
+        try:
+            if protocol == 'usb':
+                success = connection_manager.connect_usb()
+            elif protocol == 'wifi':
+                success = connection_manager.connect_wifi()
```

Pylint (3.3.7)

[error] Assigning result of a function call, where the function has no return

(E1111)